File: src/main/java/de/com/fdm/bot/commands/EchoCommand.java
package de.com.fdm.bot.commands;

import org.springframework.stereotype.Component;

@Component
public class EchoCommand implements Command {
    public String execute(Parameters params) {
        if (params.getMsg().length() < 5) {
            return "";
        }

        return params.getMsg().substring(5);
    }
}

File: src/main/java/de/com/fdm/bot/commands/EchoCommand.kt
package de.com.fdm.bot.commands

import de.com.fdm.bot.Command
import de.com.fdm.bot.api.twitch.tmi.ReceiverMessage
import org.springframework.stereotype.Component

@Component
class EchoCommand : Command {
    override fun execute(receiverMessage: ReceiverMessage): String {
        if (receiverMessage.message.length < 5) {
            return ""
        }

        return receiverMessage.message.substring(5)
    }
}